tensorflow_addons/seq2seq/tests/decoder_test.py

```diff
@@ -113,7 +113,8 @@ def _decode(start_tokens, end_token):
     concrete_function = _decode.get_concrete_function(
         tf.TensorSpec([1], dtype=tf.int32), tf.TensorSpec([], dtype=tf.int32)
     )
-    converter = tf.lite.TFLiteConverter.from_concrete_functions([concrete_function])
+    converter = tf.lite.TFLiteConverter.from_concrete_functions(
```

Comment:
You need something like:
https://github.com/tensorflow/addons/blob/master/tensorflow_addons/utils/test_utils.py#L29

Reply:
You need also to fix black linting

Reply:
As the public API changed you need to wrap this alternative implementation with a condition similar to:
https://github.com/tensorflow/addons/blob/master/tensorflow_addons/utils/test_utils.py#L29

Generally we maintain a compatibility with the latest 3 TF releases (>= 2.5.x).